PLBenchmarks/edges.py

```diff
@@ -0,0 +1,174 @@
+"""
+edges.py
+Functions and classes for handling the perturbation edges.
+"""
+import os
+import pandas as pd
+import numpy as np
+import yaml
+
+import PLBenchmarks
+import PLBenchmarks.ligands
+import PLBenchmarks.targets
+
+
+class edge:
+    """
+    Store and convert the data of one perturbation ("edge") in a :py:class:`pandas.Series`.
+
+    :param d: :py:class:`dict` with the edge data
+    :return: None
+    """
+
+    def __init__(self, d: dict):
+        """
+        Initialize edge class from a dictionary
+
+        :param d: :py:class:`dict` with the edge data
+        :return: None
+        """
+        self._data = pd.Series(d)
+        self._name = None
+
+    def addLigData(self, ligs):
+        """
+        Adds data from ligands to :py:class:`~PLBenchmarks:edges.edge`. Molecule images and the affinity difference are added.
+
+        :param ligs: :py:class:`PLBenchmarks:ligands:ligandSet` class of the same target
+        :return: None
+        """
+        l0 = None
+        l1 = None
```

Comment:
```suggestion
        ligand0 = None
        ligand1 = None
```
For Python code, I would recommend longer but more descriptive variable names. I'm highlighting one pair of variables here, but there are many other instances throughout

Reply:
Changed to more descriptive names in commit https://github.com/openforcefield/PLBenchmarks/pull/3/commits/b30b011e2d57fb8baeb8035ca82ae3782325d832